docs/assets/js/swagger.yml

```diff
@@ -2734,6 +2746,47 @@ paths:
         default:
           $ref: "#/components/responses/ServerError"
 
+  /repositories/{repository}/branches/{branch}/cherry-pick:
+    parameters:
+      - in: path
+        name: repository
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: branch
+        required: true
+        schema:
+          type: string
+    post:
+      tags:
+        - branches
+      operationId: cherryPick
+      summary: Cherry-Pick the given reference commit into the given branch
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CherryPickCreation"
+      responses:
+        204:
+          description: successful cherry-pick
+        400:
+          $ref: "#/components/responses/ValidationError"
+        401:
```

Comment:
What happens if I cherry-pick into a protected branch?

Reply:
We have 2 protection rules at the moment: `STAGING_WRITE` & `COMMIT`. Merge for example, doesn't check branch protection rules. Revert too. Maybe we should define their behavior better.